### DISCUSSION TITLE
Definition of the Table Schema

### DISCUSSION THREAD

**Discussion by drusellers** [quartznet/quartznet#2241] [Q&A]
I'm thinking it would be nice if the documentation had a description of the table schemas, specifically for me the `qrtz_simprop_triggers` table.

like what is `str_prop_3` and `int_prop_2`, and so on. If we can get something here - I'm happy to transfer over to the documentation. 

I'm also game to read the code if you can point me in the right direction.

**Comment by lahma** [comment]
This might help to understand the `qrtz_simprop_triggers`  table: https://github.com/quartznet/quartznet/blob/main/src/Quartz/Impl/AdoJobStore/SimplePropertiesTriggerPersistenceDelegateSupport.cs 

Basically it's generic storage mechanism for multiple trigger types so that there's no need to have "table per trigger type". Currently this table handles `ICalendarIntervalTrigger` and `IDailyTimeIntervalTrigger`. In ideal world you could create your own trigger type and utilize custom persistence delegate for it and use Quartz's tables and logic without the need to write full job store implementation just to get your custom trigger in and out from database.

**Reply by drusellers** [reply]
TY, a PR is incoming
